Hiim.filter

```diff
@@ -355,10 +354,10 @@ ItemDisplay[skzs]: %DOT-70%%NAME%{%TAN%Note%WHITE%: Cube Gems to %GRAY%(un)%WHIT
 // Runes
 //-----------------------------------------------------------------------------------
 // Hide El-Ort based on filter level, except in towns
-ItemDisplay[RUNE<10 FILTLVL>4 !(FILTLVL=7) !(FILTLVL=9) !(MAPID=1 OR MAPID=40 OR MAPID=75 OR MAPID=103 OR MAPID=109)]:
+ItemDisplay[RUNE<10 FILTLVL>4 !(MAPID=1 OR MAPID=40 OR MAPID=75 OR MAPID=103 OR MAPID=109)]:
```

Comment:
I think based on the push this season we should not filter rune 1-9 outside of potions only

Reply:
ill do this in a 8010